### PR TITLE
PMT #111930 & #111931 Edit Breadcrumbs

### PR DESCRIPTION
**Pull request by nbuonin**
Breadcrumbs edited to:
Home => Sessions => Session Number

Also removes "Practice Participant" text per PMT #111931

### CODE REVIEW

File: smart_sa/templates/intervention/activity.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 {% block breadcrumb-main %}
 <div id="breadcrumb-main">
-	<a href="/" onclick="if(document.completeactivity){document.completeactivity.next.value=this.href;document.completeactivity.submit();return false}">Home</a> &rarr; Counsel &rarr; Working with <b>{{participant.display_name}}</b> 
+    <a href="/" onclick="if(document.completeactivity){document.completeactivity.next.value=this.href;document.completeactivity.submit();return false}">Home</a> &rarr; <a href="/intervention/{{activity.clientsession.intervention.id}}/" onclick="if(document.completeactivity){document.completeactivity.next.value=this.href;document.completeactivity.submit();return false}">Sessions</a> &rarr; <a href="/session/{{activity.clientsession.id}}/" onclick="if(document.completeactivity){document.completeactivity.next.value=this.href;document.completeactivity.submit();return false}">{{activity.clientsession.long_title}}</a>
```
Comment (sdreher) [inline smart_sa/templates/intervention/activity.html]: As the click handlers are duplicated, now would be a good time to set up a common click function to handle these navigations.

Comment (nbuonin) [reply smart_sa/templates/intervention/activity.html]: Got it, I'll work on that.

**Comment by nbuonin** [comment]
@sdreher how it look?: https://github.com/ccnmtl/smart_sa/pull/467/commits/ffbefb1fe98876f7fcafeda9e0986e5d3c0942fe#diff-d7efa7cfdef97e77d6f7fecd3eff15a3R34

File: media/js/gamestate.js [hunk 2]
```diff
@@ -31,4 +31,19 @@
         }
     });
 
+    jQuery('#breadcrumb-main a').on('click', function() {
```
Comment (sdreher) [inline media/js/gamestate.js]: jQuery supports multiple selectors: https://api.jquery.com/multiple-selector/

Comment (nbuonin) [reply media/js/gamestate.js]: I did not know that, thanks!

**Comment by nbuonin** [comment]
This updates the breadcrumb to be:
Home => Sessions (really intervention) => Session => Activity

This was done for clarity and to better reflect the structure of the data model

**Pull request merged**